src/utils/events/event-manager.js

```diff
@@ -76,38 +76,46 @@ export default class EventManager {
    */
   on(event, handler) {
     if (typeof event === 'string') {
-      // Special handling for gestural events.
-      const recognizerEvent = EVENT_RECOGNIZER_MAP[event];
-      if (recognizerEvent) {
-        // Enable recognizer for this event.
-        this.manager.get(recognizerEvent).set({enable: true});
-
-        // Alias to a recognized gesture as necessary.
-        const eventAlias = GESTURE_EVENT_ALIASES[event];
-        if (eventAlias && !this.aliasedEventHandlers[event]) {
+      this._onOne(event, handler);
+    } else {
+      // If `event` is a map, call `on()` for each entry.
+      for (const eventName in event) {
+        this._onOne(eventName, event[eventName]);
+      }
+    }
+  }
+
+  _onOne(event, handler) {
```

Comment:
Even if we don't change the public interface, could we just call this `_addEventHandler`, or `_registerEvent`?

Would love to see a one line comment header above each internal method describing its purpose (I trust the external methods are documented in markdown).